.github/workflows/purefinance_deployment copy.yaml

```
@@ -0,0 +1,39 @@
+name: Deploy Purefinance
+
+on:
+  workflow_dispatch:
+    push:
+        branches:
+        - main
+
+jobs:
+  purefinance-build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18' 
```

Let's use the version defined in `.nvmrc`:

```suggestion
          node-version-file: .nvmrc
```